crates/winch/src/builder.rs

```diff
@@ -18,13 +18,16 @@ struct Builder {
 
 pub fn builder() -> Box<dyn CompilerBuilder> {
     let triple = Triple::host();
+    let mut flags = settings::builder();
+
+    flags
+        .enable("avoid_div_traps")
```

Comment:
`wasmtime` won't attempt to compile without this flag, even though Winch doesn't specifically use it. I haven't dug into why yet